app/api/controllers/category.py

```diff
@@ -1,6 +1,5 @@
 from flask import request
 from flask_restplus import Namespace, Resource
-from app.database.models.category import CategoryModel
```

Comment:
@epicadk You have removed this import CategoryModel line. But for now we are accessing this on line 32 and 37. 
At last, we need this line to be removed after shifting code whole API code to CategoryDAO.

Reply:
Oops missed that.